Adafruit_SGP30.cpp

```diff
@@ -156,6 +156,29 @@ boolean Adafruit_SGP30::setIAQBaseline(uint16_t eco2_base, uint16_t tvoc_base) {
   return readWordFromCommand(command, 8, 10);
 }
 
+/**************************************************************************/
+/*!
+    @brief Set the absolute humidity value [mg/m^3] for compensation to increase precision of TVOC and eCO2.
+    @param absolute_humidity A uint32_t [mg/m^3] which we will be used for compensation. If the absolute humidity is set to zero, humidity compensation will be disabled.
+    @returns True if command completed successfully, false if something went wrong!
+*/
+/**************************************************************************/
+boolean SensirionSGP30::setHumidity(uint32_t absolute_humidity) {
```

Comment:
Need to change class name to Adafruit_SGP30

Reply:
This should be the only real error. @mguntli would it take much effort to fix the class name and try again?

Reply:
Thanks for the hint!
The errors have been introduced to renaming on my own fork for the particle library..